packages/management/tanzu-framework-manager/Makefile

```diff
@@ -0,0 +1,6 @@
+# Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
If this is a newly created file, shouldn't the copyright be 2022?

Reply:
thanks for catching this.. pushed the changes.
but we need to update the package scaffolding logic to use 2022